### ISSUE TITLE
gRPC packages with '.' characters in their name break the fqrnToEndpoint logic

### ISSUE THREAD

**Issue by karlmutch** [ktr0731/evans#20]
The fqrn logic uses a final form string and splits it apart to insert the '/' before method names.  However it assumes a fixed length slice.  When a package with '.' characters is selected in the UI then a call is invoked the 'invalid FQRN format' error is raised and the call fails.

The logic should, contain a minimum length check, and the return should read something like:
```
return fmt.Sprintf("/%s/%s", strings.Join(sp[:len(sp)-1], "."), sp[len(sp)-1])
```

**Comment by ktr0731** [comment]
nice catch!  
you have forked this repository, so are you thinking fix the bug?

**Comment by karlmutch** [comment]
yes I am intending on sending a PR shortly. or will also contain changes to
allow headers

**Comment by ktr0731** [comment]
awesome!  
i am looking forward to it 😄

**Comment by ktr0731** [comment]
fixed in #21
